### ISSUE TITLE
Wrong spider example with hidden modules

### ISSUE THREAD

**Issue by vanzod** [TACC/Lmod#283]
In the spider output for a hidden module created by hiding the modulefile, the suggested example for detailed information does not include the required `--show-hidden` flag.

**Comment by rtmclay** [comment]
I assume that when doing "module spider b/3.0" and b/3.0 is hidden you want the module load statement to include "--show_hidden".  I have updated Lmod to fix this problem.  Please test Lmod 7.4.19 to see if it works for you.

**Comment by rtmclay** [comment]
This is an example of a feature and not a bug.  I have removed all the changes I made for 7.4.19 and have released 7.4.20.  Note that the source code now in 7.4.20 is the same as 7.4.18.

The spider output was and is:
```
% module --show_hidden spider b/3.0

------------------------------------------------------------
  b: b/3.0
------------------------------------------------------------

    This module can be loaded directly: module  load b/3.0
```
Even though b/3.0 is a hidden module, it can be loaded w/o the --show_hidden option.
That is the point, you need --show_hidden to show it with avail or spider but a load will find it.
Yes, adding --show_hidden won't break the command.  But one of the points of hidden modules is that a user's old job script will load a hidden module without any changes.

**Comment by rtmclay** [comment]
O.K to close?
